src/validate.py

```diff
@@ -144,3 +126,53 @@ def parse_accept_language(header, supported_langs):
 
     # If all else fails, default the language.
     return DEFAULT_LANGUAGE.lang_code
+
+def get_json_files_from_dir(path):
+
+    files_found = []
+    for root, directories, files in os.walk(path):
+        for file in files:
+            if '.json' in file:
+                files_found.append(os.path.join(root, file))
+
+    return files_found
+
+def get_entries_from_json(path, p_key, s_key):
+
+    with open(path) as json_file:
+        data = json.load(json_file)
+
+    entries = []
+
+    if p_key in data:
+        for values in data.get(p_key):
+             entries.append(values.get(s_key))
+
+    return entries
+
+def get_chapters_for_year(file):
```

Comment:
Not loving the naming of these functions. We say `_for_year` but then pass a file in. I know each year has a single file, but still it seems weird. Rename to `get_chapters_for_file(file)` or simply `get_chapters(file)`?

Reply:
have renamed them